services/brig/src/Brig/API/Public.hs

```diff
@@ -108,6 +108,7 @@ import qualified Wire.API.Routes.Public.Galley as GalleyAPI
 import qualified Wire.API.Routes.Public.LegalHold as LegalHoldAPI
 import qualified Wire.API.Routes.Public.Spar as SparAPI
 import qualified Wire.API.Routes.Public.Util as Public
+import Wire.API.Routes.Version
```

Comment:
qualify this, and call `versionSwagger` `swaggerDoc`?

Reply:
I don't like to create name clashes on purpose and resolve them with qualified imports. It makes everything more awkward and I don't see the benefit.

Reply:
I won't insist, but I think we agreed at some point (we should document these things) that imports should either be explicit or qualified, but not neither.  Doesn't have to happen now, since we're not consistent, but can you get on board with the general idea?

Reply:
Well, I disagree with the general idea, but if it's something everyone else agrees on, I'll comply.